cc.c

```diff
@@ -128,6 +128,11 @@ void prechecks(int argc, char** argv)
 			env = env + 1;
 			i += 2;
 		}
+		else if(match(argv[i], "-c"))
+		{
+			OBJECT_FILES_ONLY = TRUE;
+			i += 1;
```

Comment:
I guess here we could use more conventional `i++`?

Reply:
We could. IMO I would rather use `i += 1` for consistency with the other branches but I can change it if you want.